recipes-kernel/linux/linux-raspberrypi_5.15.bb

```diff
@@ -2,7 +2,7 @@ LINUX_VERSION ?= "5.15.24"
 LINUX_RPI_BRANCH ?= "rpi-5.15.y"
 LINUX_RPI_KMETA_BRANCH ?= "yocto-5.15"
 
-SRCREV_machine = "770d94882ac145c81af72e9a37180806c3f70bbd"
+SRCREV_machine = "ac66b3f757a3a6c84c6641356fb81c9fce3e89e9"
```

Comment:
is it still at 5.15.24 ?

Reply:
Yes https://github.com/raspberrypi/linux/blob/ac66b3f757a3a6c84c6641356fb81c9fce3e89e9/Makefile#L4